packages/portal/swagger-ui-web-component/package.json

```diff
@@ -17,7 +17,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "swagger-client": "^3.26.8",
-    "swagger-ui": "^5.1.3"
+    "swagger-ui": "~5.7.2"
```

Comment:
Any reason you're using `~` instead of `^`?

Reply:
switched back to `^`